src/main/java/me/mrCookieSlime/Slimefun/SlimefunPlugin.java

```diff
@@ -237,6 +238,7 @@ else if (getServer().getPluginManager().isPluginEnabled("CS-CoreLib")) {
             new WitherListener(this);
             new IronGolemListener(this);
             new PlayerInteractEntityListener(this);
+            new HazmatSuitListener(this);
```

Comment:
This should be renamed and gated behind an if statement (see my review comments on this Listener class)